src/portal/src/i18n/lang/zh-cn-lang.json

```diff
@@ -682,6 +688,14 @@
             "ALWAYS_ONBOARD": "Always Onboard",
             "VERIFY_CERT": "Authentication验证证书"
         },
+        "OIDC": {
+            "OIDC_PROVIDER": "OIDC 供应商",
+            "ENDPOINT": "OIDC Endpoint",
+            "CLIENT_ID": "OIDC 客户端标识",
+            "CLIENTSECRET": "OIDC CLIENT密码",
```

Comment:
you should either translate `CLIENT` in both `CLIENT_ID` and `CLIENT_SECRET`, or not translate at all.